src/NHibernate/Dialect/BitwiseNativeOperation.cs

```diff
@@ -15,28 +15,26 @@ public class BitwiseNativeOperation : ISQLFunction
 	{
 		private readonly string _sqlOpToken;
 		private readonly bool _isNot;
```

Comment:
I have not kept the renaming to `_isUnary` in order to avoid a grey area breaking change (serialization) in a release version (5.1.x). Same for constructor argument name, kept as `isNot` instead of `isUnary`.